console/views/group.py

```diff
@@ -93,12 +93,12 @@ def put(self, request, app_id, *args, **kwargs):
 
         """
         app_name = request.data.get("app_name", None)
-        app_note = request.data.get("app_note", "")
-        if app_note and len(app_note) > 2048:
+        note = request.data.get("note", "")
```

Comment:
Why change the API parameters?

Reply:
Note is used in the front-end. To unify, all use note